tests/run/cpp_method_overloading.pyx

```diff
@@ -0,0 +1,88 @@
+# mode: run
+# tag: cpp
+
+cdef extern from *:
```

Comment:
This file could use a bit of clarity about what you're actually aiming to show with all of the tests. `Derived` doesn't really seem to do anything. Is `DefinedDerived` only to test that you can override a constructor? If so would some print statements in the constructor help to prove that it was actually called?